src/uwtools/api/config.py

```diff
@@ -17,25 +17,19 @@
 
 
 def compare(
-    config_a_path: DefinitePath,
-    config_a_format: str,
-    config_b_path: DefinitePath,
-    config_b_format: str,
+    config_1_path: DefinitePath,
+    config_2_path: DefinitePath,
+    config_1_format: Optional[str] = None,
+    config_2_format: Optional[str] = None,
```

Comment:
Changed `a` -> `1` and `b` -> 2` to agree with other, related identifiers like `STR.file1path` and CLI switches like `--file-1-path`.